packages/core/package-lock.json

```diff
@@ -923,15 +923,6 @@
         "regenerator-runtime": "^0.13.2"
       }
     },
-    "@babel/runtime-corejs3": {
```

Comment:
How come so many packages were removed from package lock even though you added Deriv P2P?

Reply:
what i did was npm install inside the directory, maybe i should use lerna add --scope=core instead. let me try again